### ISSUE TITLE
WCAG.nl quick scan: fix errors

### ISSUE THREAD

**Issue by jamalv** [teamforus/general#459]
Learn more about change requests here: https://bit.ly/39CWeEE

### Requested by:
Law 👨‍⚖️ (And therefore all our clients)

### Change description
Currently we are showing up with some faults at https://wcag.nl/quickscan/h5yQqZh1bQtWpu8HCwTV

This issue is about solving the error's that are given 

- [x] Run https://wcag.nl/ quickscan on https://groningen.staging.forus.io/ 
- [x] Create dev issues for the error's @GerbenBosschieter @maxvisser  
- [x] Link to issues this epic @maxvisser 
- [x] Assign to dev's & plan in @maxvisser 
- [x] Fix issues @GerbenBosschieter @SjoerdHilhorst @maxvisser (give to rMinds if needed)
- [x] Deploy to dev, re-run scan @maxvisser @GerbenBosschieter 
- [ ] Close issue if all is green, inform @jamalv if still fails.

## Definition of done:
- [x] https://groningen.staging.forus.io/ scores without errors on the https://wcag.nl/ quickscan

**Comment by maxvisser** [comment]
Sommige best practices zou je de opbouw van de html moeten veranderen bijv het toepassen van H1, H2, H3 en/of paragraven. Maar ook bijv. <header> <footer> tags gebruiken.

Hoe ver willen we gaan met deze issue? Ik heb drie naar mijn inziens eenvoudige 'quickfixes' aangemaakt.

**Comment by maxvisser** [comment]
Can we define the scope of this issue; Do you want to resolve all issues (even the remarks?)

**Comment by jamalv** [comment]
Ah okay, I see the result has 3 categories:

Errors, Attention Points, and Best Practices. 

For the initial scope I would like the errors to be fixed at least (following the definition of done) - If we are unable to fix the errors we need an explanation of why this is impossible right now, and a plan/timeline of how we will fix it.

**Comment by maxvisser** [comment]
I think with the current quickfixes we should fix the errors atleast. 👍 Let wait until these are done and do a rescan.

**Comment by GerbenBosschieter** [comment]
@maxvisser @jamalv exactly. We are going to heck how far we can get with easy fixes to pass the scan

**Comment by GerbenBosschieter** [comment]
**Status and updates**
- The quickscan of https://wcag.nl/quickscan/3rkMvlTDufOSG1dTxAKW is reduced from 6 to 1 error.
- We have a new quickscan that is trusted by a lot of parties and the website itself also provides webshops with certifications. Also gives pretty good result: https://zelfscantoegankelijkheid.thuiswinkel.org/check_result_cpt/https-dev-forus-io-groningen/
- Lighthouse (google developer tool foor accessibility) gives us a score of 76/100 (pretty good)
- We should manually check if everything is controllable with the keyboard

After the above is done we can make the scope bigger and do more manual stuffwith screenreaders, etc etc

**Comment by maxvisser** [comment]
released
